docs/authentication-roadmap.md

```
@@ -0,0 +1,103 @@
+# Authentication roadmap
+
+This roadmap is normative. Every entry is independently grabbable. Published issue links are recorded below; PR: `[ ]`; Status reflects current work.
+
+## R0. Normative docs, decisions, diagrams, report, and issues
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)